integration_test/src/lib.rs

```diff
@@ -0,0 +1 @@
+// Nothing here...
```

Comment:
Is this file required?

Reply:
If I remove it, I get:
```
error: failed to parse manifest at `/home/v0idpwn/oss/pgmq/integration_test/Cargo.toml`
Caused by:
  can't find library `pgmq-integration-tests`, rename file to `src/lib.rs` or specify lib.path
```
Maybe there's something we can write on Cargo.toml to avoid this error?

Reply:
I think if we remove the `[lib]` section from Cargo.toml we can remove the `./src` dir and lib.rs.

Reply:
Awesome!